docs/PID tuning.md

```diff
@@ -29,6 +29,24 @@ changing slowly (so the P and I terms aren't having enough impact on reaching th
 in the correction in order to reach the target sooner. If the error is rapidly converging to zero, the D term causes the
 strength of the correction to be backed off in order to avoid overshooting the target.
 
+##TPA and TPA Breakpoint
+
+* Note that TPA is set via CLI or on the PID TUNING tab of the GUI.  tpa_breakpoint is set via CLI
+
+TPA applies a 'P' value reduction in relation to full Throttle. It is used to 'soften' the P value at extream throttle which sometimes induces oscellation in the aircraft.
```

Comment:
I would be happy to include your edits within this PR if you would like to add them.
https://github.com/dave-pitman/cleanflight/blob/dave-pitman-pid/docs/PID%20tuning.md